versions.props

```diff
@@ -6,6 +6,7 @@ com.google.guava:guava = 21.0
 com.palantir.baseline:* = 0.24.0
 com.palantir.safe-logging:* = 1.4.0
 org.slf4j:slf4j-api = 1.7.24
+gradle.plugin.com.palantir:gradle-circle-style = 1.2.0-rc6
```

Comment:
we should release 1.2.0 once we have confidence it works. We shouldn't let all the features there just rot